stacktrace.go

```diff
@@ -199,25 +197,14 @@ func fileContext(filename string, line, context int) ([][]byte, int) {
 	return lines[start:end], idx
 }
 
-var trimPaths []string
-
 // Try to trim the GOROOT or GOPATH prefix off of a filename
+// We don't use `go/build` here as it will look at the _current_ values,
```

Comment:
while you're here, add a period to the above sentence and an empty comment line between these two paragraphs.